docs/Common-Issues.md

```diff
@@ -113,3 +113,7 @@ Following a macOS upgrade it may be necessary to reinstall the Xcode Command Lin
 xcode-select --install
 brew upgrade
 ```
+
+### Local build failures
+
+Build failures on one machine, but not similarly equipped machines (check issues for other projects too), can indicate that the install is borked. If `brew doctor` doesn’t, reinstalling may help. That is, `brew bundle dump`, [uninstall](https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew), [reinstall](https://docs.brew.sh/Installation), and `brew bundle install`.
```

Comment:
What if, instead, we add a more general item for "things just seem broken or messed up" (i.e. borked but I'm struggling to find the right wording for some reason) that just says "if you're having problems you can try to reinstall Homebrew to see if that fixes the problem. To do so easily, run ...

I agree we don't necessarily need to be worried about build-failures as building form source isn't supported, but it may be worth having instructions for easily reinstalling Homebrew somewhere and that's really the information that you are trying to add.

Here's my suggestion which doesn't have to be the final version if you have other
```suggestion
### Other local issues

If your Homebrew installation gets messed up (and fixing the issues found by `brew doctor` doesn't solve the problem), reinstalling Homebrew may help to reset to a normal state. To easily reinstall Homebrew, use [Homebrew Bundle](https://github.com/Homebrew/homebrew-bundle) to automatically restore your installed formulae and casks. To do so, run `brew bundle dump`, [uninstall](https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew), [reinstall](https://docs.brew.sh/Installation), and run `brew bundle install`.
```

This is also something that may belong in the FAQ.